### PR TITLE
View Multiple Buckets

### PR DESCRIPTION
**Pull request by eddiesarevalo**
## 🗒️ Summary
Multiple buckets can now be viewed at once.

## ⚙️ Test Data and/or Report
Set up normally as in the readme if you have access to MCP and AWS.

A faster way is to switch to this branch. Then run `npm run build`, then run "python -m SimpleHTTPServer 9001" on the root directory of the project.
Open a browser to localhost:9001 then verify everything runs. (You might need a CORS extension for your browser to get around CORS blocks).

## ♻️ Related Issues
- fixes https://github.com/NASA-PDS/s3-browser-cloudfront/issues/22

### CODE REVIEW

**Comment by nutjob4life** [comment]
`python -m SimpleHTTPServer 9001` only works in Python 2, which reached end-of-life on January 1st, 2020!

The modern equivalent would be `python3 -m http.server`

File: src/index.html [hunk 1]
```diff
@@ -1,120 +1,175 @@
...
-                    </button>
-                    <div class="collapse navbar-collapse" id="preview">
-                        <div class="d-flex">
-                            <input class="form-control" type="search" placeholder="Filter..." aria-label="Filter">
-                        </div>
-                    </div>
-                </div>
-            </div>
+      // Single Page Apps for GitHub Pages
+      // MIT License
```
Comment (nutjob4life) [inline src/index.html:16]: I thought we were using Apache License v2? 🤔

File: src/index.html [hunk 2]
```diff
@@ -1,120 +1,175 @@
...
+                top: 0px;
+                left: 0px;
+                opacity: 1;
+                transition: filterBrightness 900ms cubic-bezier(0.4, 0, 0.2, 1)
+                    0s,
+                  filterSaturate 1200ms cubic-bezier(0.4, 0, 0.2, 1) 0s,
+                  opacity 600ms cubic-bezier(0.4, 0, 0.2, 1) 0s;
+              "
+            />
+            <p class="footerLinksTitle">PLANETARY DATA SYSTEM</p>
```
Comment (nutjob4life) [inline src/index.html:102]: Should this be `Planetary Data System` and the CSS class `footerLinksTitle` should have `text-transform: uppercase`?

**Review by nutjob4life — CHANGES_REQUESTED** [review]
Looks pretty good otherwise!

**Review by jordanpadams — APPROVED** [review]

**Comment by jordanpadams** [comment]
@eddiesarevalo is this build expected to fail on Node.js 16.x and 18.x?

**Comment by eddiesarevalo** [comment]
@jordanpadams it works on node v16 and v18. I can add an nvmrc file though so that nvm could take care to use the latest v20

**Comment by anilnatha** [comment]
@jordanpadams The Node.js tests breaking are something we need to address in other repos too. For this repo, I think we should remove the Node.js 14.x and 16.x tests as they don't provide any value. Both 14 and 16 versions are no longer supported by Node.js

**Comment by jordanpadams** [comment]
@anilnatha @eddiesarevalo copy. will remove 14 and 16, for v18, how should we update your branch build so we build successfully?

```
    steps:
    - uses: actions/checkout@v4
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v4
      with:
        node-version: ${{ matrix.node-version }}
        cache: 'npm'
        token: ${{secrets.ADMIN_GITHUB_TOKEN || github.token}}
    - run: npm ci
    - run: npm run build --if-present
    - run: npm test
```
It looks like it is failing on the npm test because there is no test file.

**Pull request merged**